go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/vmware-tanzu/community-edition
```

Comment:
I don't think having a top level go mod file will work with our current multiple go modules paradigm.

Won't this cause circular imports on _lower_ modules?

My recommendation is to put it in a lower dir, maybe a `pkg` dir?

Reply:
It does work fine. Even though the source structure contains the other modules, the resulting module itself does not. So go mod resolution works as expected.

Reply:
A `pkg` dir could make sense though. But that typically would still have a root `go.mod` file.

Reply:
If it works than 👍🏼 ! And it makes sense to have a top level module that defines how our other lower level modules should implement that plugin descriptor